mod.config

```diff
@@ -90,10 +94,10 @@ PACKAGING_WINDOWS_REGISTRY_KEY="OpenRAExampleMod"
 PACKAGING_WINDOWS_LICENSE_FILE="./COPYING"
 
 # The git tag to use for the AppImage dependencies.
-PACKAGING_APPIMAGE_DEPENDENCIES_TAG="20200222"
+PACKAGING_APPIMAGE_DEPENDENCIES_TAG="20200328"
 
 # Space delimited list of additional files/directories to copy from the engine directory
-# when packaging your mod. e.g. "./mods/modcontent" or "./mods/d2k/OpenRA.Mods.D2k.dll"
+# when packaging your mod. e.g. "./mods/modcontent"
 PACKAGING_COPY_ENGINE_FILES=""
```

Comment:
```suggestion
PACKAGING_COPY_ENGINE_FILES="./IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP"
```

Reply:
`global mix database.dat` as well? (it is listed in the NSIS installer)

Reply:
`IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP` is part of the internal engine data (just like the glsl files or lua wrappers), and is already copied by the engine install function.

Unfortunately, due to upstream limitations, `global mix database.dat` is too even though it really shouldn't be. The way we load and package this file needs to be changed upstream before we can fix that in the SDK.